lib/shoryuken/logging.rb

```
@@ -11,8 +15,7 @@ def tid
       end
 
       def context
-        c = Thread.current[:shoryuken_context]
-        c ? " #{c}" : ''
+        Fiber.current.shoryuken_context&.then{|context| " #{context}"}
```

RuboCop (1.76.1)

[convention] Space missing to the left of {.

(Layout/SpaceBeforeBlockBraces)